bin/s3rver.js

```diff
@@ -63,3 +63,17 @@ new S3rver(program).run(function(err, host, port) {
     console.log("now listening on host %s and port %d", host, port);
   }
 });
+
+s3rver.s3Event.subscribe(function(event) {
+  // eslint-disable-next-line no-console
+  console.log(event);
+});
+
+s3rver.s3Event
+  .filter(function(event) {
+    return event.Records[0].eventName == "ObjectCreated:Copy";
+  })
+  .subscribe(function(event) {
+    // eslint-disable-next-line no-console
+    console.log(event);
```

Comment:
Same comment as above – logs should be configurable, so we don't force users to accept noisy logs.

Reply:
I have removed the code from this file. With this feature, you can only subscribe to events when you run s3rver programatically as you need to subscribe to s3rver.s3Event observable. Not sure how you would do that if you run s3rver from command line. Any suggestions? Pass a function as parameter?

Reply:
You could add another command line argument above in this file, and create the subscription only if the argument is set. Would need to add a little more documentation to make the feature accessible to users.

Reply:
that would not help because users would not get event notification in their project. May be we could use remote observable (https://github.com/jbaudanza/rxremote) to subscribe to server over websocket. Something may be for next release.